web/templates/inbox_row.hbs

```diff
@@ -0,0 +1,41 @@
+{{#if is_stream}}
+<div id="stream_header_{{stream_id}}" class="inbox_header {{#if is_hidden}}hidden_by_filters{{/if}}" tabindex="0" data-stream-id="{{stream_id}}" style="background: {{stream_header_color}};">
+    <div class="left_part_wrapper">
+        <div class="collapsible-button"><i class="zulip-icon zulip-icon-arrow-down toggle-inbox-header-icon"></i></div>
+        <div class="left_part">
+            <div tabindex="0" class="inbox_header_name">
+                <span id="stream_sidebar_privacy_swatch_{{stream_id}}" class="stream-privacy filter-icon" style="color: {{stream_color}}">
```

Comment:
This is almost certainly a duplicate element ID. I think there's a few others, like `stream_header_{{stream_id}}`, that seem suspiciously like they may not be unique -- prefixing names of unique elements for this view with `inbox-` should help.

Reply:
Yeah, fixed.